frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelines.cy.ts

```diff
@@ -54,100 +54,6 @@ describe('Pipelines', () => {
     pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
   });
 
-  it('Configure pipeline server when data connection exist', () => {
```

Comment:
We need to keep this -- if we have to rewrite this for our new S3 Connection Types -- so be it, but this is a good test to have for "copying from".